rotkehlchen/tests/unit/accounting/evm/test_balancer.py

```diff
@@ -117,40 +190,100 @@
+        balance=Balance(amount=FVal('1675.495956074927519908')),
+        location_label=USER_ADDRESS,
+        notes='Return 1675.495956074927519908 BPT to a Balancer v1 pool',
+        counterparty=CPT_BALANCER_V1,
+        identifier=None,
+        extra_data={'withdrawal_events_num': 2},
+    ), EvmEvent(
+        event_identifier=EVM_HASH_2,
+        sequence_index=95,
+        timestamp=TIMESTAMP_2_MS,
+        location=Location.ETHEREUM,
+        event_type=HistoryEventType.WITHDRAWAL,
+        event_subtype=HistoryEventSubType.REMOVE_ASSET,
+        asset=A_DAI,
+        balance=Balance(amount=FVal('124999.999999999999867')),
+        location_label=USER_ADDRESS,
+        notes='Receive 124999.999999999999867 DAI after removing liquidity from a Balancer v1 pool',  # noqa: E501
+        counterparty=CPT_BALANCER_V1,
+        identifier=None,
+        extra_data=None,
+    ), EvmEvent(
+        event_identifier=EVM_HASH_2,
+        sequence_index=97,
+        timestamp=TIMESTAMP_2_MS,
+        location=Location.ETHEREUM,
+        event_type=HistoryEventType.WITHDRAWAL,
+        event_subtype=HistoryEventSubType.REMOVE_ASSET,
+        asset=A_USDC,
+        balance=Balance(amount=FVal('124850.618089')),
+        location_label=USER_ADDRESS,
+        notes='Receive 124850.618089 USDC after removing liquidity from a Balancer v1 pool',  # noqa: E501
+        counterparty=CPT_BALANCER_V1,
+        identifier=None,
+        extra_data=None,
+    ),
+]
+
 
 @pytest.mark.parametrize('should_mock_price_queries', [True])
 @pytest.mark.parametrize('default_mock_price_value', [ONE])
 def test_balancer_multiple_deposit(accountant: 'Accountant', ethereum_inquirer: 'EvmNodeInquirer'):
     """Test that the default accounting settings for balancer are correct"""
     pot = accountant.pots[0]
     transactions = pot.transactions
-    curve_accountant = Balancerv1Accountant(
+    balancer_v1_accountant = Balancerv1Accountant(
         node_inquirer=ethereum_inquirer,
         msg_aggregator=ethereum_inquirer.database.msg_aggregator,
     )
-    curve_settings = curve_accountant.event_settings(pot)
-    transactions.tx_event_settings.update(curve_settings)
+    balancer_settings = balancer_v1_accountant.event_settings(pot)
+    transactions.tx_event_settings.update(balancer_settings)
     events_iterator = iter(DEPOSIT_ENTRIES)
     for event in events_iterator:
         pot.transactions.process(event=event, events_iterator=events_iterator)
 
     expected_events = [
         ProcessedAccountingEvent(
             type=AccountingEventType.TRANSACTION_EVENT,
-            notes='Deposit 131578.947368421052491563 mUSD to a Balancer v1 pool',
+            notes='Deposit 131578.947368421052491563 DAI to a Balancer v1 pool',
             location=Location.ETHEREUM,
             timestamp=TIMESTAMP_1_SECS,
-            asset=Asset('eip155:1/erc20:0xe2f2a5C287993345a840Db3B0845fbC70f5935a5'),
+            asset=A_DAI,
```

Comment:
why changing the asset on the tests? You took another tx as reference to make things easier?

Reply:
I just wanted to use something easier.